--- libtool.m4  2018-09-25 10:50:23.682668594 +0100
+++ libtool.m4  2018-09-25 10:51:30.942420511 +0100
@@ -4695,6 +4695,13 @@
    _LT_TAGVAR(lt_prog_compiler_pic, $1)='-KPIC'
    _LT_TAGVAR(lt_prog_compiler_static, $1)='-static'
         ;;
+      *flang)
+      # Flang compiler
+   _LT_TAGVAR(lt_prog_compiler_wl, $1)='-Wl,'
+   _LT_TAGVAR(lt_prog_compiler_pic, $1)='-fPIC -DPIC'
+   _LT_TAGVAR(lt_prog_compiler_static, $1)='-static'
+       ;;
+  
       # icc used to be incompatible with GCC.
       # ICC 10 doesn't accept -KPIC any more.
       icc* | ifort*)
